index.js

```diff
@@ -6,26 +6,173 @@ const openNav = () => {
-const closeButton = document.querySelector('#close-btn');
-function Display() {
-  modalPopup.style.display = 'block';
+const workpage = document.querySelector('#workpage');
+for (let i = 0; i < projects.length; i += 1) {
+  workpage.innerHTML += `
+  <div class="project-card flex column">
+  <img src="${projects[i].img}" alt="project card image" />
+  <div class="prject-card-info flex column">
+    <h2 class="project-title">${projects[i].name}</h2>
+    <div class="project-meta flex">
+      <p class="bold">${projects[i].des[0]}</p>
+      <div class="round"></div>
+      <p>${projects[i].des[1]}</p>
+      <div class="round"></div>
+      <p>${projects[i].des[2]}</p>
+    </div>
+    <div class="project-note">
+      <p>
+      ${projects[i].description}
+      </p>
+    </div>
+    <div class="project-tag">
+      <ul class="project-skills flex">
+        <li>${projects[i].technologies[0]}</li>
+        <li>${projects[i].technologies[1]}</li>
+        <li>${projects[i].technologies[2]}</li>
+      </ul>
+    </div>
+    <button type="button" id="btn-${i}">See Project</button>
+  </div>
+  `;
 }
-function closePopup() {
-  modalPopup.style.display = 'none';
+
+const modal = document.getElementById('modal1');
+
+for (let j = 0; j < projects.length; j += 1) {
+  const seeBtn = document.querySelectorAll(`[id=btn-${j}]`);
+  seeBtn.forEach((bt) => {
+    bt.addEventListener('click', () => {
+      modal.style.display = 'block';
+      modal.innerHTML = `
+      <div class="popup-container block-popup">
+      <div id="popup-modal" class="pop-content">
+        <div class="headline-popup">
+          <h1>${projects[j].name}</h1>
+          <a href="#" role="button" onclick="closePopup()">
+            <div class="popup-clsbtn">x</div>
+          </a>
+        </div>
+      </div>
+      <ul class="small-tags">
+        <li class="canopy-sm-tag">${projects[j].des[0]}</li>
+        <li>
+          <div class="round-popup"></div>
+        </li>
+        <li class="second-sm-tag">${projects[j].des[1]}</li>
+        <li>
+          <div class="round-popup"></div>
+        </li>
+        <li class="second-sm-tag">${projects[j].des[2]}</li>
+      </ul>
+      <div class="popup-image-container">
+        <img src="${projects[j].img}" alt="pop-image" />
+      </div>
+      <div class="buttom-content-container">
+        <div class="paragraph-description">
+          <p class="paragraph-description-desktop">
+            ${projects[j].desc1}
+          </p>
+        </div>
+        <div class="action-content">
+          <div class="popup-tags">
+            <ul class="first">
+            <li>${projects[j].technologies[0]}</li>
+            <li>${projects[j].technologies[1]}</li>
+            <li>${projects[j].technologies[2]}</li>
+            </ul>
+            <ul class="second">
+            <li>${projects[j].technologies1[0]}</li>
+            <li>${projects[j].technologies1[1]}</li>
+            <li>${projects[j].technologies1[2]}</li>
```

Comment:
- [Optinal] Kindly I think it better to create a new variable to loop through the technology lists and use it as it is not always three👍👌